animation_nodes/nodes/object/collection_operations.py

```diff
@@ -0,0 +1,33 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+
+class CollectionOperationsNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_CollectionOperationsNode"
+    bl_label = "Collection Operations"
+
+    useObjectList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("Collection", "Collection", "collection", defaultDrawType = "PROPERTY_ONLY")
+        self.newInput(VectorizedSocket("Object", "useObjectList",
+            ("Object", "object"),
```

Comment:
You can set the `defaultDrawType`—or other properties—like this:
```python
("Object", "object", dict(defaultDrawType = "PROPERTY_ONLY"))
```

Reply:
Done in 827cae1.